# Handel the exception 
arr = ["mango","apple","orange"]
try:
    for i in range(5) :
        print(arr[i])
except :
    print("Exception Handled ")
print("this line has to be mandatorily exception ")